data/posts-router.js

```diff
@@ -0,0 +1,82 @@
+const Posts = require('./db.js');
+
+const express = require('express');
```

Comment:
Express was missing from your `package.json` dependencies. There might be commits that need to be pushed.